src/test/java/Page_Objects/SongListPage.java

```diff
@@ -114,7 +116,7 @@ public void goToPlaylist(String playlistName) {
     public void expectPlaylistToBeSelected(String playlistName) {
         String playlistSelectedAttribute = playlistElement(playlistName).getAttribute("data-selected");
 
-        Assertions.assertEquals(playlistSelectedAttribute, "true");
+        Assertions.assertEquals("true", playlistSelectedAttribute, "Expected playlist is not selected");

```

Comment:
it's always a good idea to add what exactly was expected and what is the actual result. E.g. this error message could be
```
"Expected playlist " + playListName + " to be selected but it is not. data-selected=" + playlistSelectedAttribute + " instead"
```

Reply:
Ok, i got it. Thanks for tip :)